app/views/provider/admin/dashboards/_ajax_load.html.slim

```diff
@@ -1,4 +1,23 @@
 javascript:
   document.addEventListener('DOMContentLoaded', function () {
-    dashboardWidget(#{json widget.path})
+    let widgetPath = #{json widget.path}
+    let serviceId = #{json widget.params[:service_id]}
+
+    if (serviceId === null) { // Audience Widget
+      dashboardWidget(widgetPath)
+    } else {
+      let value = window.localStorage['toggle:service_' + serviceId]
+      if (value && !JSON.parse(value)['is-closed']) {
+        dashboardWidget(widgetPath)
+      }
+
+      let container = document.getElementById('service_' + serviceId)
+      let title = container.querySelector('.DashboardSection-toggle')
+      title.addEventListener('click', function () {
```

Comment:
this should be inside an `if` verifying that `title` has actually found. Because when the connection with backend fails, this title doesn't exist and it will raise an error trying the `addEventListener` over an element that does not exist  😄 An example of this is locally in development 😄

Reply:
Good catch, thanks!